<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XLEX正则解析</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;</string>
       </property>
       <property name="text">
        <string>请输入正则表达式（支持0-9a-zA-Zε[]()?*+|）:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;</string>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="styleSheet">
      <string notr="true">font-size: 28px;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="generateNFA">
       <property name="styleSheet">
        <string notr="true">font-size:24px;</string>
       </property>
       <property name="text">
        <string>生成NFA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateDFA">
       <property name="styleSheet">
        <string notr="true">font-size:24px;</string>
       </property>
       <property name="text">
        <string>生成DFA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateMFA">
       <property name="styleSheet">
        <string notr="true">font-size:24px;</string>
       </property>
       <property name="text">
        <string>生成MFA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateCode">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;</string>
       </property>
       <property name="text">
        <string>生成词法分析程序</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>开始状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="startState">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>结束状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="endState">
       <property name="styleSheet">
        <string notr="true">font-size: 24px;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
